services/softlayer_virtual_guest.go

```diff
@@ -380,7 +380,7 @@ func (slvgs *softLayer_Virtual_Guest_Service) AttachEphemeralDisk(instanceId int
 		return err
 	}
 
-	order := datatypes.SoftLayer_Product_Order{
+	order := datatypes.SoftLayer_Ephemeral_Disk_Order{
```

Comment:
@maximilien 
Thanks for your comments,  all the testcases including `AttachEphemeralDisk` and `PlaceEphemeralDiskOrder` had been changed accordingly.